src/Psalm/CallMap.php

```diff
@@ -2703,7 +2703,7 @@
 'gmp_gcdext' => ['array', 'a'=>'', 'b'=>''],
 'gmp_gcd' => ['GMP', 'a'=>'', 'b'=>''],
 'gmp_hamdist' => ['int', 'a'=>'', 'b'=>''],
-'gmp_import' => ['GMP', 'data'=>'string', 'word_size'=>'int', 'options'=>'int'],
+'gmp_import' => ['GMP', 'data'=>'string', 'word_size'=>'?int', 'options'=>'?int'],
```

Comment:
Shouldn't this be `'word_size=' => 'int', 'options=' => 'int'` ? These params are optional, but documentation does not imply they are nullable (rather, they have a default int value).